aws-go-s3-folder/Pulumi.yaml

```diff
@@ -1,3 +1,9 @@
 name: aws-go-s3-folder
 runtime: go
-description: A static website hosted on AWS S3.
+description: A static website hosted on AWS S3
+template:
+  description: A static website hosted on AWS S3
```

Comment:
Based on the way the CLI works currently and for historical reasons, we need to add a template description if we want a description to show up when you run `pulumi new https://github.com/pulumi/examples` to get a list of examples, e.g.:

```
$ pulumi new https://github.com/pulumi/examples/tree/justin/button-only
Please choose a template:
> aws-go-s3-folder                                A static website hosted on AWS S3
  aws-js-s3-folder                                A static website hosted on AWS S3
  aws-js-s3-folder-component                      A static website hosted on AWS S3
  aws-js-sqs-slack                                Post to Slack for each SQS message!
  aws-js-webserver                                Basic example of an AWS web server accessible over HTTP
  aws-js-webserver-component                      Basic example of an AWS web server accessible over HTTP
  aws-py-stepfunctions                            Basic example of AWS Step Functions in Python
  aws-py-webserver                                Basic example of an AWS web server accessible over HTTP (in Python!)
  aws-ts-airflow                                  AWS RDS and Airflow example
  aws-ts-eks                                      EKS cluster example
  aws-ts-resources                                A Pulumi program that demonstrates creating various AWS resources
  aws-ts-ruby-on-rails                            A Ruby on Rails stack using a single EC2 instance with a local MySQL database for storage
  aws-ts-serverless-raw                           Basic example of a serverless AWS application
  aws-ts-static-website                           Static website example
  aws-ts-stepfunctions                            Basic example of AWS Step Functions
  azure-js-webserver                              Basic example of an Azure web server accessible over HTTP
  azure-ts-aks-helm                               Create an Azure Kubernetes Service (AKS) cluster and deploy a Helm Chart into it
  azure-ts-aks-mean                               A simple Deployment running nginx, exposed to the Internet with a Service
  azure-ts-appservice                             Creates Azure App Service with SQL Database and Application Insights
  azure-ts-functions                              Azure Functions example
  cloud-js-api                                    A simple HTTP endpoint that returns the number of times a route has been hit
  cloud-js-containers                             NGINX container example
  cloud-js-httpserver                             A simple HTTP server that returns the number of times a route has been hit
  cloud-js-thumbnailer                            A video thumbnail extractor using serverless functions and containers
  cloud-js-thumbnailer-machine-learning           A video thumbnail extractor using serverless functions, containers, and AWS Rekognition
  cloud-js-twitter-athena                         Analyze tweets in AWS Athena
  cloud-ts-url-shortener                          URL shortener sample
  cloud-ts-url-shortener-cache                    URL shortener with cache
  cloud-ts-url-shortener-cache-http               URL shortener with cache and HttpServer
  cloud-ts-voting-app                             Voting app that uses containers
  gcp-js-webserver                                Basic example of an Google Cloud web server accessible over HTTP
  gcp-ts-functions                                GCP Functions example
  gcp-ts-gke                                      A Google Kubernetes Engine (GKE) cluster, with canary deployment
  kubernetes-ts-configmap-rollout                 A Deployment that mounts a ConfigMap; updating the data in the ConfigMap triggers a rollout of the Deployment
  kubernetes-ts-exposed-deployment                A simple Deployment running nginx, exposed to the Internet with a Service
  kubernetes-ts-helm-wordpress                    A minimal installation of the latest version of the Wordpress Helm chart
  kubernetes-ts-jenkins                           Jenkins deployment on Kubernetes
  kubernetes-ts-nginx                             Example of a Kubernetes Stateless Application Deployment, using Nginx
  kubernetes-ts-s3-rollout                        A simple Deployment obtaining data from S3
  kubernetes-ts-sock-shop                         The standard Weaveworks Sock Shop demo based on https://github.com/microservices-demo/microservices-demo
  kubernetes-ts-staged-rollout-with-prometheus    Staged application rollout gated by Prometheus checks
  twilio-ts-component                             A small example that builds a Pulumi component for Twilio Programable SMS
```

Without the template description, we'd just show the folder names:

```
$ pulumi new https://github.com/pulumi/examples
Please choose a template:
> aws-go-s3-folder                                
  aws-js-s3-folder                                
  aws-js-s3-folder-component                      
  aws-js-sqs-slack                                
  aws-js-webserver                                
  aws-js-webserver-component                      
  aws-py-stepfunctions                            
  aws-py-webserver                                
  aws-ts-airflow                                  
  aws-ts-eks                                      
  aws-ts-resources                                
  aws-ts-ruby-on-rails                            
  aws-ts-serverless-raw                           
  aws-ts-static-website                           
  aws-ts-stepfunctions                            
  azure-js-webserver                              
  azure-ts-aks-helm                               
  azure-ts-aks-mean                               
  azure-ts-appservice                             
  azure-ts-functions                              
  cloud-js-api                                    
  cloud-js-containers                             
  cloud-js-httpserver                             
  cloud-js-thumbnailer                            
  cloud-js-thumbnailer-machine-learning           
  cloud-js-twitter-athena                         
  cloud-ts-url-shortener                          
  cloud-ts-url-shortener-cache                    
  cloud-ts-url-shortener-cache-http               
  cloud-ts-voting-app                             
  gcp-js-webserver                                
  gcp-ts-functions                                
  gcp-ts-gke                                      
  kubernetes-ts-configmap-rollout                 
  kubernetes-ts-exposed-deployment                
  kubernetes-ts-helm-wordpress                    
  kubernetes-ts-jenkins                           
  kubernetes-ts-nginx                             
  kubernetes-ts-s3-rollout                        
  kubernetes-ts-sock-shop                         
  kubernetes-ts-staged-rollout-with-prometheus    
  twilio-ts-component    
```

I'm not a huge fan of the redundant project and template descriptions. Now that we're no longer doing the `${DESCRIPTION}` string replacing, I can update the CLI to just display the project description when there is no template description, and then we can update the Pulumi.yaml files here to remove the template description. I'll open a bug.